src/IQFeed.CSharpApiClient/Streaming/Level1/Messages/IUpdateSummaryMessage.cs

```diff
@@ -20,6 +20,8 @@ public interface IUpdateSummaryMessage
         double Close { get; }
         string MessageContents { get; }
         string MostRecentTradeConditions { get; }
+
+        // TODO: drop this for the next MAJOR version, once we migrate to the new solution
```

Comment:
mark it deprecated?

Reply:
sure, let's decide if we are OK adding the new fields to `IUpdateSummaryMessage` or we still want to keep the new interface.
I will make this change